Tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php

```diff
@@ -24,9 +25,8 @@ public function setUp()
 
     public function testFormTwigTemplate()
     {
-        $twigExtension = $this->getContainer()->get('twig');
-        $twigExtension->initRuntime();
-        $renderer = $twigExtension->getExtension('form')->renderer;
+        /** @var TwigRendererInterface $renderer */
+        $renderer = $this->getContainer()->get('twig')->getExtension('form')->renderer;
```

Comment:
this property is no longer public in symfony 3.2 . any idea how we can achieve this in a better way?